### PR TITLE
Remove node test runner for blueprint-test-helpers

### PR DESCRIPTION
**Pull request by binhums**
ember-cli-blueprint-test-helpers was upgraded to 0.12.0 in https://github.com/rwjblue/ember-cli-template-lint/commit/9ce14333144053acddf8a8ea047dc445d1f87c6a.

This PR updates ember-cli-template-lint to the new way of running these node tests.

### CODE REVIEW

File: node-tests/acceptance/broccoli-test.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe('broccoli-template-linter', function() {
     return cleanupBuilders();
   });
 
-  it('uses provided generateTestFile to return a test file', function() {
+  it('uses provided generateTestFile to return a test file', function(done) {
```
Comment (rwjblue) [inline node-tests/acceptance/broccoli-test.js]: Returning a promise should be the same as using the `done` pattern here (AFAIK).

Comment (binhums) [inline node-tests/acceptance/broccoli-test.js]: I was getting timeouts with my local environment, perhaps something else is going on.

Comment (binhums) [inline node-tests/acceptance/broccoli-test.js]: Yes, still getting this issue. Should see if mocha has changed.

**Comment by rwjblue** [comment]
Seems like node 4 and 6 are passing, but not 0.12.  Haven't looked at what is going on yet though...

**Comment by binhums** [comment]
I'm running 0.12 locally.

**Comment by binhums** [comment]
@rwjblue looks like it fails on whatever the first test that runs in the file is. I.e. if I duplicate the first test it fails and then passes.

Is there something that takes a while to initialize here?

**Comment by rwjblue** [comment]
@binhums - Mind rebasing this to see if it is passing now?

**Pull request closed without merging**